core/tests/swfs/avm1/as_broadcaster/output.txt

```diff
@@ -0,0 +1,13 @@
+true
```

Comment:
It would be very helpful if you can print what it is you're about to test, so that if this breaks in the future we can see why instead of opening the .fla and figuring out which line it was.

So like instead of:
```actionscript
trace(broadcaster.addListener(listener2));
broadcaster.broadcastMessage("event1");
```
You can do:
```actionscript
trace("// broadcaster.addListener(listener2)");
trace(broadcaster.addListener(listener2));
trace("");

trace("// broadcaster.broadcastMessage(\"event1\")");
broadcaster.broadcastMessage("event1");
trace("");
```

I know it's more verbose but it makes it so much easier to tell what's wrong when it doesn't work!